vendor.mod

```diff
@@ -46,8 +46,8 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/gddo v0.0.0-20190904175337-72a348e765d2
 	github.com/golang/protobuf v1.5.3
-	github.com/google/go-cmp v0.5.9
-	github.com/google/uuid v1.3.0
+	github.com/google/go-cmp v0.6.0
+	github.com/google/uuid v1.5.0
```

Comment:
Oh! I recall I had a PR to update these, but at the time we decided it wasn't critical for 25.0, so moved it to draft; I just did a quick rebase, and can move it out of draft if we need this one;

- https://github.com/moby/moby/pull/46980